nats/README.md

```diff
@@ -8,32 +8,49 @@ This document defines how to describe NATS-specific information on AsyncAPI.
 
 Current version is `0.1.0`.
 
-
 <a name="server"></a>
 
 ## Server Binding Object
 
 This object MUST NOT contain any properties. Its name is reserved for future use.
 
+##### Fixed Fields
 
+| Field Name                                                       |  Type  | Description                                                                         |
+|------------------------------------------------------------------|:------:|-------------------------------------------------------------------------------------|
+| <a name="serverBindingObjectPayload"></a>`payload`               | string | Defines the payload data type. Can be either `binary`, `JSON` or `string` (default) |
```

Comment:
Isn't this covered by the contentType field on the message object?

Reply:
Good catch, I used the [typescript client](https://github.com/nats-io/nats.ts#connect-options) as example and they define it when connecting to the server. But yea based on the [NATS documentation](https://docs.nats.io/nats-protocol/nats-protocol#pub) which should be able to use the existing contentType field. Gonna remove it 👍